### PR TITLE
Internal doc: unpacked directory formats

### LINKED ISSUE #170: Describe the format/location/... of generated environments in the doc
There should be a page for advanced users describing exactly what the VMs/directories are built, if people want to reuse them directly without reprounzip.

### PR DESCRIPTION
**Pull request by remram44**
Closes #170

### CODE REVIEW

**Comment by remram44** [comment]
@fchirigati do you want to take a look?

File: docs/unpacked-format.rst [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+..  _unpacked-directory:
+
+The `directory` Unpacker
+========================
+
+The experiment directory contains:
+
+* The original configuration file ``config.yml``.
+* The pickle file ``.reprounzip``.
```
Comment (fchirigati) [inline docs/unpacked-format.rst]: @remram44: is there anything worth mentioning that this file stores for the directory unpacker?

File: docs/unpacked-format.rst [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+Unless ``--dont-bind-magic-dirs`` is specified when unpacking, the special directories ``/dev``, ``/dev/pts``, and ``/proc`` are mounted with ``mount -o bind`` from the host.
+Also, if ``/bin/sh`` or ``/usr/bin/env`` weren't both packed, a static build of `busybox <https://busybox.net/>`__ is downloaded and put under ``/bin/busybox``, and the missing binaries are created as symbolic links pointing to busybox.
+
+Should you require a shell inside the experiment environment, you can use::
+
+    chroot root/ /bin/sh
+
+..  _unpacked-vagrant:
+
+The `vagrant` Unpacker
```
Comment (fchirigati) [inline docs/unpacked-format.rst]: @remram44: any reason for not showing the experiment structure as you did in the directory and chroot unpackers? Same for the docker unpacker.

**Comment by fchirigati** [comment]
@remram44: done!

File: docs/unpacked-format.rst (hunk 1 above)
Comment (remram44) [inline docs/unpacked-format.rst]: No, nothing specific is stored there, although I should probably add a paragraph about how input files are handled for all the unpackers.

File: docs/unpacked-format.rst (hunk 2 above)
Comment (remram44) [inline docs/unpacked-format.rst]: I can add that!

**Pull request merged**